### DISCUSSION TITLE
[Feature] adding {user} option in storage  template

### DISCUSSION THREAD

**Discussion by YoadMatza** [immich-app/immich#14867] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

to have an option to set the storage template by user as well so will be able to achieve user/album/file name or user/y/m/file name 
for example {{user}}/{{#if album}}{{album}}{{else}}{{y}}/{{MM}}{{/if}}/{{filename}}

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Files are already stored in separate folders per user. You can set the folder name in the user settings.

**Reply by YoadMatza** [reply]
yeap, missed it :)
